packages/core/src/extensions/DraggableBlocks/DraggableBlocksPlugin.ts

```diff
@@ -556,23 +540,30 @@ export class BlockMenuView<BSchema extends BlockSchema> {
       unfreezeMenu: () => {
         this.menuFrozen = false;
       },
+      getReferenceRect: () => {
+        if (!this.menuOpen) {
+          return this.lastPosition;
```

Comment:
lastPosition is the `blockContentBoundingBox`, but if we don't use lastPosition we create a new DOMRect. This looks inconsistent